src/components/generic/Button.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+const StyledButtons = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+`;
+
+type buttonInfo = {
+  name: string;
+  clickCallback: () => void;
+}
+
+const Buttons = ({inputButtons}: {inputButtons: buttonInfo[]}) => {
```

Comment:
I like the idea of making generic components, but these are too generic so you're not saving much in how its implemented. Your 4 timers have the same 4 buttons with different callbacks. Your Buttons component should instead be something like `TimerControls`, and you would simply pass in the callbacks. The labels are shared across all of them. Additionally, you should try to add some constraints in the actions a user can take based on the application state. For example, Start should be disabled/hidden if the timer is running, Pause should be disabled/hidden if the timer is not running. Was a bit confused about the difference between Start and Resume. Was not expecting Start to reset the timer back to its initial state. Lastly, you're missing the Fast Forward button which was part of the requirements.